File: aoj/volume100/java/A10009.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double r = scan.nextDouble();
        double area = Math.pow(r, 2) * Math.PI;
        double circumference = 2 * r * Math.PI;
        System.out.printf("%.6f %.6f\n", area, circumference);
    }
}

File: aoj/volume100/ocaml/A10009.ml
let pi =
    4.0 *. atan 1.0

let area r =
    r *. r *. pi

let circumference r =
    2.0 *. r *. pi

let solve r =
    Printf.printf "%.6f %.6f\n" (area r) (circumference r)

let () =
    Scanf.scanf "%f" solve
